client/gutenberg/extensions/editor-notes/style.scss

```diff
@@ -1,17 +1,4 @@
-.wp-block-a8c-editor-notes {
-	border: 2px solid gray;
-	position: relative;
-	padding: 32px 6px 6px;
-	background-color: #c0e7ff;
-
-	.is-selected {
-		padding-top: 6px;
-	}
-}
-
-.editor-notes__editor-indicator {
-	position: absolute;
-	top: 0;
-	left: 8px;
-	font-style: italic;
+.wp-block-a8c-editor-notes .card {
```

Comment:
Is this here to ensure `card` is correctly overridden?

See https://github.com/Automattic/wp-calypso/pull/26933#discussion_r214826802